vshard/storage/init.lua

```diff
@@ -2901,8 +2901,9 @@ local function rebalancer_service_f(service)
                 goto continue
             end
         end
-        log.info('Rebalance routes are sent. Schedule next wakeup after '..
-                 '%f seconds', consts.REBALANCER_WORK_INTERVAL)
+        log.info('Next rebalancer routes were sent: %s. Schedule next ' ..
```

Comment:
Nit: sounds incorrect gramatically) Let's better say `The following rebalancer routes were sent`, or you can just leave as it was in order not to change the existing tests)